x-pack/plugins/observability_solution/profiling/server/lib/setup/get_has_setup_privileges.ts

```diff
@@ -31,8 +32,11 @@ export async function getHasSetupPrivileges({
         },
       },
       kibana: [
-        securityPluginStart.authz.actions.api.get(`${FLEET_PLUGIN_ID}-all`),
-        securityPluginStart.authz.actions.api.get(`${INTEGRATIONS_PLUGIN_ID}-all`),
+        securityPluginStart.authz.actions.api.get(ApiOperation.Read, `${FLEET_PLUGIN_ID}-all`),
```

Comment:
@cauemarcondes `Read` should be sufficient here, right?

Reply:
Not really. The Profiling setup needs to be performed by an administrator, and it needs to create some fleet integrations.

Reply:
@cauemarcondes Thank you! I changed `read` to `manage`